quinn-proto/src/assembler.rs

```diff
@@ -89,6 +118,9 @@ impl Assembler {
 
     pub(crate) fn insert(&mut self, offset: u64, bytes: Bytes) {
         self.data.push(Chunk { offset, bytes });
+        if self.data.len() - self.defragmented > 32 {
```

Comment:
Should have a brief comment describing the arbitrariness of this constant.

Reply:
Fair, will add something.